lib/src/data/simulations.dart

```diff
@@ -88,6 +90,14 @@ class Simulations with ChangeNotifier {
         infoLink: 'https://en.wikipedia.org/wiki/Lissajous_curve',
         fav: _favorites[5],
       ),
+      SimulationCard(
+        id: 6,
+        simulationName: 'Epicycloid Curve',
```

Comment:
This needs to get more specific as we discussed

Reply:
The simulation name isn't completely visible if "(pencil of lines)" is added. How should I go about it?